Description: Upstream changes introduced in version 6.5-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 basex (6.5-1) unstable; urgency=low
 .
   * Initial release (Closes: #588216)
 .
 The person named in the Author field signed this changelog entry.
Author: Alexander Holupirek <alex@holupirek.de>
Bug-Debian: http://bugs.debian.org/588216

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- a/pom.xml
+++ b/pom.xml
@@ -84,12 +84,9 @@
         <configuration>
           <archive>
             <manifest>
-              <mainClass>org.basex.BaseXGUI</mainClass>
+              <mainClass>org.basex.BaseX</mainClass>
               <packageName>org.basex</packageName>
             </manifest>
-            <manifestEntries>
-              <SplashScreen-Image>img/start.png</SplashScreen-Image>
-            </manifestEntries>
           </archive>
         </configuration>
       </plugin>
